src/app/(main)/profile/myActivitiesStatus/page.tsx

```
@@ -9,58 +10,80 @@ import {
-  const [selectedDate, setSelectedDate] = useState<Date | null>(null);
-  const [selectedTab, setSelectedTab] = useState<'신청' | '승인' | '거절'>('신청');
-  const [selectedTime, setSelectedTime] = useState('14:00 - 15:00');
-  const [, setVisibleCount] = useState(2);
+  // 📅 날짜 관련 상태
+  const [date, setDate] = useState<Date | null>(new Date()); // 캘린더에서 선택된 월
+  const [selectedDate, setSelectedDate] = useState<Date | null>(null); // 모달에서 보여줄 특정 날짜
+
+  // 🎯 모달 내 필터링 상태
+  const [selectedTab, setSelectedTab] = useState<'신청' | '승인' | '거절'>('신청'); // 모달 탭
+  const [selectedTime, setSelectedTime] = useState('14:00 - 15:00'); // 선택된 시간대
+  const [, setVisibleCount] = useState(2); // 표시할 예약 개수 (미사용)
```

ESLint

[error] 'setVisibleCount' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)